DOCUMENTATION.md

```diff
@@ -0,0 +1,115 @@
+# Provider Documentation
+
+## Building and Reviewing
+
+The documentation is built from templates stored in the `templates`
+folder, along with examples stored in the `examples` folder. All of
+this content is copied to the `docs` folder, with the templates fully
+rendered, when the documentation is built.
```

Comment:
I believe no comma is necessary.

Reply:
Yeah, I'm going to ignore that recommendation, this sort of phrase does need to be delimited by commas :)